scripts/microbe_stage/microbe.as

```diff
@@ -186,6 +191,8 @@ class MicrobeComponent : ScriptComponent{
     ObjectID microbeEntity = NULL_OBJECT;
 
     Float3 queuedMovementForce = Float3(0, 0, 0);
+
+    int totalHexCountCache = 0;
```

Comment:
A comment for this variable would be nice. Explaining what this is used (fast size check).

Reply:
Added in new commit